centreon/www/install/php/Update-25.01.0.php

```diff
@@ -17,3 +18,76 @@
  * For more information : contact@centreon.com
  *
  */
+
+require_once __DIR__ . '/../../../bootstrap.php';
+require_once __DIR__ . '/../../class/centreonLog.class.php';
+
+$centreonLog = CentreonLog::create();
+
+// error specific content
+$versionOfTheUpgrade = 'UPGRADE - 25.01.0: ';
+$errorMessage = '';
+
+$addColumnToResourcesTable = function (CentreonDB $pearDBO) use (&$errorMessage): void {
+    $errorMessage = 'Unable to add column flapping to table resources';
+    if (! $pearDBO->isColumnExist('resources', 'flapping')) {
+        $pearDBO->exec(
+            <<<'SQL'
+                ALTER TABLE `resources`
+                ADD COLUMN `flapping` TINYINT(1) NOT NULL DEFAULT 0
```

Comment:
comments ?

Reply:
will be handled in the PR for View by Host